scripts/initial_map.json

```diff
@@ -1,115 +1,82 @@
 {
```

Comment:
This file does not have to be changed, I think you ran scripts/compile_current.js